electrumx/server/block_processor.py

```diff
@@ -256,13 +256,23 @@ def __init__(self, env, daemon, notifications):
             self.header_mc.truncate(self.height + 1)
         await self.prefetcher.reset_height(self.height)
 
-    async def reorg_hashes(self, count):
+    async def reorg_hashes(self, count=None):
```

Comment:
I don't think the default =None is needed - can we drop it and also in both instances of calc_reorg_hashes?

Reply:
Done